cmd/install.go

```diff
@@ -107,20 +93,50 @@ func getUserList(sysUsers v1beta1.SystemUser) []string {
 	return values
 }
 
-func getCmdArgs(role string) []string {
-	var args []string
+func getCmdArgs(role string) (args []string) {
+	args = append(args, role)
+
+	if dataDir != "" {
+		args = append(args, "--data-dir", dataDir)
+	}
 
 	if role == "server" {
-		args = append(args, "server")
+		if serverToken != "" {
+			args = append(args, "--token-file", serverToken)
```

Comment:
shouldn't `serverToken` actually be changed to `tokenFile`?

Reply:
The variable is inherited from here https://github.com/k0sproject/k0s/blob/main/cmd/server.go#L59 so it is already parsed.

Reply:
I believe this is the argument you are looking for: https://github.com/k0sproject/k0s/blob/main/cmd/server.go#L53

Reply:
Fixed it, can you please re-review it.